tests/benchmarks/parallel_advantage/parallel_cos_2x.amber

```diff
@@ -0,0 +1,83 @@
+#!amber
```

Comment:
Did you mean to add these amber script files?

Reply:
not in this pr :)